src/sdk/pynni/nni/metis_tuner/metis_tuner.py

```diff
@@ -370,12 +370,12 @@ def _selection(self, samples_x, samples_y_aggregation, samples_y,
                         temp_improvement = threads_result['expected_lowest_mu'] - lm_current['expected_mu']
     
                         if next_improvement > temp_improvement:
-                            logger.infor("DEBUG: \"next_candidate\" changed: \
-                                            lowest mu might reduce from %f (%s) to %f (%s), %s\n" %\
-                                            lm_current['expected_mu'], str(lm_current['hyperparameter']),\
-                                            threads_result['expected_lowest_mu'],\
-                                            str(threads_result['candidate']['hyperparameter']),\
-                                            threads_result['candidate']['reason'])
+                            # logger.info("DEBUG: \"next_candidate\" changed: \
```

Comment:
Is it better to remove these commented lines?

Reply:
sure~